ert/tracker/evaluator_tracker.py

```diff
@@ -198,7 +216,7 @@ def request_termination(self):
             return
 
         with create_ee_monitor(self._ee_con_info) as monitor:
-            for e in monitor.track():
+            for _ in monitor.track():
                 monitor.signal_cancel()
                 break
```

Comment:
```suggestion
            monitor.signal_cancel()
```

I _think_ that should work fine now. We had a bug in the monitor where you needed to call `track()` before interacting with it, but that was fixed in https://github.com/equinor/ert/commit/bf60d4910943534df4d1a1fb97fba5314ece124f